inf-ruby.el

```diff
@@ -103,17 +103,24 @@ returns a string."
                            inf-ruby-implementations))
   :group 'inf-ruby)
 
+(defcustom inf-ruby-wrapper-command ""
+  "Launcher command pattern to format the auto-detected command.
+Useful for running the shell in another host or a container (such as Docker). Must include %s."
+  :type 'string
+  :group 'inf-ruby)
+
 (defun inf-ruby--irb-command ()
   (let ((command "irb --prompt default -r irb/completion --noreadline"))
     (when (inf-ruby--irb-needs-nomultiline-p)
       (setq command (concat command " --nomultiline")))
     command))
 
 (defun inf-ruby--irb-needs-nomultiline-p (&optional with-bundler)
-  (let* ((output (car (last (apply #'process-lines
-                                   (append
-                                    (when with-bundler '("bundle" "exec"))
-                                    '("irb" "-v"))))))
+  "Check if IRB needs the --nomultiline argument.
+WITH-BUNDLER, the command is wrapped with `bundle exec'."
+  (let* ((command (format (if (string-empty-p inf-ruby-wrapper-command) "%s" inf-ruby-wrapper-command)
```

Comment:
If we're doing a different thing based on the default value, how about we make the default `nil`?

Then this form can be replaced with just: `(format (or inf-ruby-wrapper-command "%s"))`